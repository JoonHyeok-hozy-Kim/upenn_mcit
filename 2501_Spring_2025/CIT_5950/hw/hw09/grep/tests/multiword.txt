./test_files/Hello.txt Hello World!
./test_files/Byte.txt Goodbye world Goodbye
./test_files/mutual_aid.txt a comrade a comrade
./test_files/war_and_peace.txt I concluded. war and peace
./test_files/mutual_aid.txt the a and a they
