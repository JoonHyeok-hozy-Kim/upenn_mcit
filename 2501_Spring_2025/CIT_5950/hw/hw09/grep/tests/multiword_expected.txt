./test_files/Hello.txt: [Hello, World!]
Hello World!
./test_files/Byte.txt: [Goodbye, world, Goodbye]
./test_files/mutual_aid.txt: [a, comrade, a, comrade]
upon a comrade in case of need. One of them had fallen upon its back in
solicitude for their wounded, and do not abandon a wounded comrade
nest which a comrade is building, or even steals from it a few sprays of
away an injured comrade enjoys a well-merited popularity.(30) So also
./test_files/war_and_peace.txt: [I, concluded., war, and, peace]
war and the peace that had been concluded. "Yes, I have been much
./test_files/mutual_aid.txt: [the, a, and, a, they]
necessary, they transport the corpse to a suitable soft ground; and they
their intelligence; they grasp the new conditions in a moment, and act
bands. They choose a place in the forest to stay there, and thence they
a flock and fly away if the robber is an erne; they plunge into the lake
if it is a falcon; or they raise a cloud of water-dust and bewilder the
coming from the forest, and maintain that, if heard, they forbode a
morality, and they prefer to cast a doubt upon the exactitude of
ruin and become a slave through not paying the due compensation: they
of a hungry child; they must feed it, and so they do. "When the school
